src/allmydata/storage_client.py

```diff
@@ -761,9 +765,8 @@ def _make_storage_system(self, node_config, config, ann):
                 # able to get the most up-to-date value.
                 self.get_rref,
             )
-        except AnnouncementNotMatched:
-            # Nope.
-            pass
+        except AnnouncementNotMatched as e:
+            print('No plugin for storage-server "{nickname}" from plugins: {plugins}'.format(nickname=ann.get("nickname", "<unknown>"), plugins=e.args[0]))
```

Comment:
`twisted.python.log.msg` (unless you want to figure out `twisted.logger` here) please.  I know `print` goes to the same place but it goes through an adaption layer that precludes any structure (as a result - no structure in this message) and is really only intended as a debugging convenience.

Reply:
`log.msg` is also un-structured (or at least doesn't respect `{}` formatting), so I did `twisted.logger.Logger`